sync-secrets-heroku/Pulumi.yaml

```diff
@@ -0,0 +1,75 @@
+name: sync-secrets-heroku
+description: sync secrets from 1Password to Heroku
+runtime: yaml
+config:
+  'pulumi:tags': {value: {'pulumi:template': yaml}}
+  github:owner:
+        value: conda-forge
+variables:
+  autotick-bot-gh-token:
+    fn::invoke:
+      function: onepassword:getItem
+      arguments:
+        title: autotick-bot-gh-token 
+        vault: pulumi
+  cf-admin-github-token: # GH_TOKEN
+    fn::invoke:
+      function: onepassword:getItem
+      arguments:
+        title: cf-admin-github-token
+        vault: pulumi
+  cf-webservices-feedstock-private-key:
+    fn::invoke:
+      function: onepassword:getItem
+      arguments:
+        title: cf-webservices-feedstock-private-key
+        vault: pulumi
+  cf-webservices-token:
+    fn::invoke:
+      function: onepassword:getItem
+      arguments:
+        title: cf-webservices-token
+        vault: pulumi
+  cf-webservices-private-key:
+    fn::invoke:
+      function: onepassword:getItem
+      arguments:
+        title: cf-webservices-private-key
+        vault: pulumi
+  heroku-only-staging-binstar-token:
+    fn::invoke:
+      function: onepassword:getItem
+      arguments:
+        title: heroku-only-staging-binstar-token
+        vault: pulumi
+  prod-binstar-token:
+    fn::invoke:
+      function: onepassword:getItem
+      arguments:
+        title: prod-binstar-token
+        vault: pulumi
+
```

Comment:
This newline is odd.

Reply:
We can add a yaml formatter in pre-commit!